### ISSUE TITLE
brew tap fails

### ISSUE THREAD

**Issue by tkralphs** [dpo/homebrew-openblas#87]
Hey @dpo, I cannot currently tap this.
```
Running `brew update --preinstall`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 4 formulae.

==> Tapping dpo/openblas
Cloning into '/mnt/c/Users/tkral/Documents/.linuxbrew/Homebrew/Library/Taps/dpo/homebrew-openblas'...
remote: Enumerating objects: 251, done.
remote: Total 251 (delta 0), reused 0 (delta 0), pack-reused 251
Receiving objects: 100% (251/251), 61.10 KiB | 1.22 MiB/s, done.
Resolving deltas: 100% (134/134), done.
Error: Invalid formula: /mnt/c/Users/tkral/Documents/.linuxbrew/Homebrew/Library/Taps/dpo/homebrew-openblas/Formula/octave.rb
octave: undefined method `devel' for #<Class:0x0000000002e7d360>
Error: Invalid formula: /mnt/c/Users/tkral/Documents/.linuxbrew/Homebrew/Library/Taps/dpo/homebrew-openblas/Formula/r.rb
r: Unsupported special dependency :java
Error: Cannot tap dpo/openblas: invalid syntax in tap!
```
I saw similar errors when trying to update my old OS X box. But somehow, the tests on `homebrew-coinor` are passing, although this is a dependency. Is everything working for you?

**Comment by dpo** [comment]
@tkralphs I haven't touched this tap in years. It's really fallen behind. I can't tap it either. The reason seems to be that some brew syntax has changed in the meantime, and the tap doesn't pass the bar.

Some of the formulae here have much improved versions in [`dpo/mumps-jl`](https://github.com/dpo/homebrew-mumps-jl).

**Comment by tkralphs** [comment]
Fair enough, but it's currently still a dependency of coinor, right? I thought you were talking about updating it? Otherwise, I guess we need to remove the dependency? I'm a little confused how others are managing to install cbc then. Maybe they already have this tap and haven't tried to upgrade?

**Comment by dpo** [comment]
You're right, we need to sort that out. That's partly why it's useful to have CI. Unfortunately, it looks like the CircleCI setup has been broken for a while now.

It looks like we just need a sequential MUMPS formula, right?

**Comment by tkralphs** [comment]
Depends on what you mean by "need". Mumps is actually only used by Clp, now that I'm thinking about it, so it doesn't even need to be a dependency of Cbc. For Clp, it's optional, not sure how much it helps. But it certainly doesn't hurt. As an aside, for Cbc, we could add a dependency on nauty, which it uses for symmetry detection. There is already a homebrew package for it.
